tools/ds-identify

```diff
@@ -268,6 +268,8 @@ read_fs_info() {
                 dev=${line#DEVNAME=};;
             LABEL=*) label="${line#LABEL=}";
                      labels="${labels}${line#LABEL=}${delim}";;
+            LABEL_FATBOOT=*) label="${line#LABEL_FATBOOT=}";
+                     labels="${labels}${line#LABEL_FATBOOT=}${delim}";;
```

Comment:
This is another option. Slightly shorter... I dont have a strong feeling either way.  I woudl like to request that you only do the '#' operation once by using the 'label' variable on line 272 in the assignment to 'labels' in 273. I am not sure why 269-270 didn't do that.

```diff
diff --git a/tools/ds-identify b/tools/ds-identify
index 071cdc0ca..4e5700fc6 100755
--- a/tools/ds-identify
+++ b/tools/ds-identify
@@ -266,8 +266,9 @@ read_fs_info() {
                     isodevs="${isodevs},${dev}=$label"
                 ftype=""; dev=""; label="";
                 dev=${line#DEVNAME=};;
-            LABEL=*) label="${line#LABEL=}";
-                     labels="${labels}${line#LABEL=}${delim}";;
+            LABEL=*|LABEL_FATBOOT=*)
+                label="${line#*=}";
+                labels="${labels}${label}${delim}";;
             TYPE=*) ftype=${line#TYPE=};;
             UUID=*) uuids="${uuids}${line#UUID=}$delim";;
         esac
```